build-tools/automation/azure-pipelines.yaml

```diff
@@ -277,13 +282,139 @@ stages:
         artifactName: Test Results - MSBuild Smoke - Linux
 
     - template: yaml-templates/fail-on-issue.yaml
-      
+
 - template: yaml-templates/stage-msbuild-tests.yaml
-      
+
 - template: yaml-templates/stage-msbuild-emulator-tests.yaml
   parameters:
     usesCleanImages: ${{ parameters.macTestAgentsUseCleanImages }}
 
+- stage: maui_tests
+  displayName: MAUI Tests
```

Comment:
It's nice this doesn't actually take that long:

![image](https://github.com/xamarin/xamarin-android/assets/840039/7424ef53-428d-4eef-ab0e-f32eaa1fcfda)